.pre-commit-config.yaml

```diff
@@ -35,13 +39,13 @@ repos:
       entry: cmd.exe /c "scons checkPot -j 4"
       language: system
       pass_filenames: false
-      types: [python, pofile]
```

Comment:
This would never run when python files would be changed. Note that you need `types_or` to handle multiple types, using `types` expects a file to match all the types.